python/lsst/sims/maf/metrics/calibrationMetrics.py

```diff
@@ -121,6 +121,7 @@ def __init__(self, metricName='properMotion',
         super(ProperMotionMetric, self).__init__(col=cols, metricName=metricName, units=units,
                                                  badval=badval, **kwargs)
         # set return type
+        self.mjdCol = mjdCol
```

Comment:
It means I had to be sure to manually set up any stackers so that they would use the correct column names. We might want to throw warnings when stackers get automatically launched so it's easy to track down what ones might be needed.